cmd/wallet-web/src/pages/Store.vue

```diff
@@ -167,13 +170,24 @@ export default {
   methods: {
     ...mapGetters(['getCurrentUser']),
     ...mapGetters('agent', { getAgentInstance: 'getInstance' }),
+    toggleDetails(credential) {
+      credential.showDetails = !credential.showDetails;
+    },
     prepareCards: function (data) {
-      data.verifiableCredential.map((vc) => {
-        const manifest = this.getManifest(vc);
-        const record = this.getCredentialDisplayData(vc, manifest);
-        this.records.push(record);
-      });
-      console.log('records', JSON.stringify(this.records, null, 2));
+      // If only one credential then show details by default
+      if (data.verifiableCredential.length === 1) {
+        const manifest = this.getManifest(data.verifiableCredential[0]);
+        const credential = this.getCredentialDisplayData(data.verifiableCredential[0], manifest);
+        this.processedCredentials.push({ ...credential, showDetails: true });
+      } else {
+        data.verifiableCredential.map((vc) => {
+          const manifest = this.getManifest(vc);
+          const credential = this.getCredentialDisplayData(vc, manifest);
+          this.processedCredentials.push({ ...credential, showDetails: false });
+        });
+      }
+
+      console.log('processedCredentials', JSON.stringify(this.processedCredentials, null, 2));
```

Comment:
left over?

Reply:
No it was there before as well but i can delete

Reply:
Let's delete the console